kubeval/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.10
+
+ENV KUBEVAL_VERSION=0.14.0 \
+    KUBECTL_VERSION=1.16.2
+
+RUN apk add --no-cache \
+      curl \
+      wget \
+      git
+
+RUN wget https://github.com/instrumenta/kubeval/releases/download/0.14.0/kubeval-linux-amd64.tar.gz && \
+    tar xf kubeval-linux-amd64.tar.gz && \
+    mv kubeval /usr/local/bin && \
+    rm -rf kubeval-linux-amd64.tar.gz && \
+    \
+    curl -L \
```

Comment:
Is this the only reason why `curl` is installed? In that case I think we can avoid it. The first download is done with `wget`, and that one already follows reditrects

Reply:
fixed.

Reply:
Great!